docs/package.json

```diff
@@ -44,11 +44,12 @@
     "@mui/styles": "^5.14.1",
     "@mui/system": "^5.14.1",
     "@mui/types": "^7.2.4",
-    "@mui/x-data-grid": "6.10.0",
+    "@mui/x-data-grid": "6.10.1",
     "@mui/x-data-grid-generator": "6.10.0",
-    "@mui/x-data-grid-pro": "6.10.0",
-    "@mui/x-date-pickers": "6.10.0",
-    "@mui/x-date-pickers-pro": "6.10.0",
+    "@mui/x-data-grid-premium": "6.10.1",
```

Comment:
TLDR: Adding `@mui/x-data-grid-premium` to dependencies solved the issue.

`@mui/x-data-grid-premium` is a dependency of `@mui/x-data-grid-generator`.
Last week we released a new version of `@mui/x-data-grid-premium`, but didn't release `@mui/x-data-grid-generator`.
Because `@mui/x-data-grid-premium` was a transitive dependency, we ended up having `@mui/x-data-grid-pro` and `@mui/x-data-grid-premium` of different versions 🙃

Reply:
> Last week we released a new version of @mui/x-data-grid-premium, but didn't release @mui/x-data-grid-generator

Nice but I didn't get this. Even in the previous versions (prior to `6.10.1`), `@mui/x-data-grid-generator` was not released right?

Reply:
> in the previous versions (prior to 6.10.1), @mui/x-data-grid-generator was not released right as seen in the Changelog?

Oh, we don't list it in the changelog, but it was released in v6.10.0: https://unpkg.com/browse/@mui/x-data-grid-generator@6.10.0/